filer/templates/admin/filer/folder/directory_listing.html

```diff
@@ -130,7 +130,7 @@
                             <div class="navigator-breadcrumbs-folder-name-wrapper">
                                 <span class="navigator-breadcrumbs-folder-name">
                                     <span class="navigator-breadcrumbs-folder-name-inner">
-                                        {% if folder.name != _("Unsorted Uploads") %}
+                                        {% if not folder.is_unsorted_uploads %}
                                             {{ folder.name }}
                                         {% else %}
                                             {% trans "Unsorted Uploads" %}
```

Comment:
why not just ``{{ folder.name }}``? You can change that to be ``_("Unsorted Uploads")``. I don't think the ``if`` is needed.